src/main/java/frc/robot/RobotContainer.java

```diff
@@ -35,17 +35,17 @@
  * commands, and button mappings) should be declared here.
  */
 public class RobotContainer {
-  private Joystick m_driveJoystick;
-  private Joystick m_operatorJoystick;
-
-  private DriveTrain m_driveTrain;
-  private Shooter m_shooter;
-  private Climber m_climber;
-  private Limelight m_limelight;
-  private PowerDistributionPanel m_pdp;
-  private Revolver m_revolver;
-  private Intake m_intake;
-  private Turret m_turret;
+  private final Joystick m_driveJoystick;
+  private final Joystick m_operatorJoystick;
+  
```

Comment:
Nitpick: Please remove trailing spaces.

VS Code actually has a setting to remove all trailing code whenever you save, if you'd like to automate this.